src/ethereum/EnigmaContractAPIBuilder.js

```diff
@@ -83,7 +92,8 @@ class EnigmaContractAPIBuilder {
       this.config = defaultsDeep(config, this.config);
     }
     if (this.config.enigmaContractABI === undefined) {
-      const EnigmaContractJson = require(path.join(this.config.truffleDirectory, 'build/contracts/Enigma.json'));
+      const rawdata = fs.readFileSync((path.join(this.config.truffleDirectory, 'build/contracts/Enigma.json')));//require(path.join(truffleDirectory, 'build/contracts/Enigma.json'));
```

Comment:
Why the change from `require` to `fs.readFileSync`?

Reply:
Because it is only loaded once and if the file changes (which happens between tests) it doesn't updated..